packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -898,6 +898,7 @@ async function handler(
     conferenceCredentialId,
     destinationCalendar,
     hideCalendarNotes: eventType.hideCalendarNotes,
+    hideCalendarEventDetails: eventType.hideCalendarEventDetails,
```

Comment:
We need to set `hideCalendarEventDetails` in other places as well. I'd say everywhere where we do `eventManager.create(evt)`, for example in `handleConfirmation`